core/src/main/java/org/opensearch/sql/executor/QueryId.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.sql.executor;
+
+import lombok.Getter;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.opensearch.sql.executor.execution.AbstractPlan;
+
+/**
+ * Query id of {@link AbstractPlan}.
+ */
+public class QueryId {
+  /**
+   * Query id.
+   */
+  @Getter
+  private final String queryId;
+
+  /**
+   * Generate {@link QueryId}.
+   * @return {@link QueryId}.
+   */
+  public static QueryId queryId() {
+    return new QueryId(RandomStringUtils.random(10, true, true));
```

Comment:
May be a good idea to call this `randomQueryId` or `randomId` or similar.  

Like UUID does: https://docs.oracle.com/javase/7/docs/api/java/util/UUID.html#randomUUID()

Reply:
queryid is associate with each query. user could cancel the query with queryid. are you suggesting randomId as var name?